src/js/components/NavTabsComponent.jsx

```diff
@@ -1,21 +1,38 @@
 var classNames = require("classnames");
 var React = require("react/addons");
 
+var DeploymentEvents = require("../events/DeploymentEvents");
+var DeploymentStore = require("../stores/DeploymentStore");
+var DeploymentActions = require("../actions/DeploymentActions");
+
 var NavTabsComponent = React.createClass({
   displayName: "NavTabsComponent",
 
   propTypes: {
     activeTabId: React.PropTypes.string.isRequired,
+    activeDeployments: React.PropTypes.number,
     className: React.PropTypes.string,
     tabs: React.PropTypes.array.isRequired
   },
 
+  getInitialState: function () {
+    return {
+      activeDeployments: this.props.activeDeployments || 0
+    };
+  },
+
   getDefaultProps: function () {
     return {
       className: ""
     };
   },
 
+  componentWillMount: function () {
+    DeploymentStore.on(DeploymentEvents.CHANGE, () => {
+      this.setState({activeDeployments: DeploymentStore.deployments.length});
+    });
+  },
```

Comment:
Should we also add a 
```
componentDidMount: function() {
  DeploymentActions.requestDeployments();
}
```
?

The way I see it, is that we are sure that Marathon is performing that request within the poll interval, so we would be doubling it here. Agree @aldipower ?